docs/architecture/adr-050-sign-mode-textual.md

```diff
@@ -51,11 +51,71 @@ This also prevents users signing over any hashed transaction data (fee, transact
 
 We propose to maintain functional tests using bijectivity in the SDK.
 
-### 2. Only ASCII 32-127 characters allowed
+### 2. UTF-8 characters allowed, but signing devices MAY convert them before display
 
```

Comment:
Here's a relevant thread on why we chose ASCII in the first place: https://github.com/cosmos/cosmos-sdk/pull/10701#discussion_r831311549

Reply:
Probably should explain this in a paragraph and not directly in a markdown header?